packages/studio-ui/src/web/components/Layout/TopNav/cloud/deploy.tsx

```diff
@@ -0,0 +1,33 @@
+import { Spinner, SpinnerSize } from '@blueprintjs/core'
+import { lang } from 'botpress/shared'
+import React from 'react'
+
+type Status = 'pending' | 'in-progress' | 'completed' | 'failed'
+
+const Step = (props: { status: Status; text: string }): JSX.Element => {
+  const { status, text } = props
+  const chars: { [s in Status]: string | JSX.Element } = {
+    pending: '⌛',
+    'in-progress': <Spinner size={SpinnerSize.SMALL} />,
+    completed: '✅',
+    failed: '🚫'
+  }
+
+  return (
+    <div style={{ display: 'flex', justifyContent: 'space-between', gap: '10px' }}>
```

Comment:
as a matter of consistency, move styles in .scss file please 🙏 .
This comments applies to everywhere you write style=

Reply:
Done in cd999da